include/picongpu/traits/attribute/DampedWeighting.hpp

```diff
@@ -64,7 +64,7 @@ namespace picongpu
              * @param multiplier weighting multiplier, values of < 1 result in damping, > 1 result in boosting
              */
             template<typename T_Particle>
-            HDINLINE void dampWeighting(T_Particle& particle, float_X const multiplier)
+            HDINLINE void dampWeighting(T_Particle& particle, float_X minWeighting, float_X const multiplier)
```

Comment:
new functionality introduced - is this intened for this PR?

Reply:
Ah, just changed interface to avoid extracting it later - fine

Reply:
yes if I read the value here directly from the param than I need to include all particle param files. If you provide it via signature I decrease include dependencies.

A PR for this change will not gain anything and will avoid merging this for at least 4 days.